src/master/master.cpp

```diff
@@ -2816,7 +2816,13 @@ void Master::_accept(
 
           // Add task.
           if (pending) {
-            _offeredResources -= addTask(task_, framework, slave);
+            Resources taskResources; Resources ports;
```

Comment:
This removes network ports as a consumable resource. Because every container gets one or more unique network interfaces, there's never a port conflict to worry about. This simplified networking is one of the many advantages of Joyent's container-native infrastructure.